src/Common/src/Interop/Windows/sspicli/GlobalSSPI.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Net
 {
     internal static class GlobalSSPI
     {
-        internal static SSPIInterface SSPIAuth = new SSPIAuthType();
```

Comment:
@stephentoub Would it be simple to add your test to a new PerformanceTests folder for S.N.Security?